frontend/packages/dev-console/src/components/import/advanced/AdvancedSection.tsx

```diff
@@ -83,12 +93,12 @@ const List: React.FC<AdvancedSectionProps> = ({ appResources, values }) => {
 
 const AdvancedSection: React.FC<AdvancedSectionProps> = ({ values, appResources }) => {
   const { t } = useTranslation();
-
+  const formPage = window.location.pathname.split('/')[1];
```

Comment:
We can get the formType from `values.formType` instead of getting it from the URL. Values props is available in `List` component so no need to pass the extra `formPage` prop to the List component,

Reply:
formType is supposed to hold create or edit values to indicate if the form is being used to create new resources or edit the old ones. Its not for the form name. Didn't wanted to it override it that is why followed this approach.